token/cli/src/main.rs

```diff
@@ -2771,6 +2949,32 @@ fn app<'a, 'b>(
                         .help("Specify the specific token account address to sync"),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::EnableRequiredTransferMemos.into())
+                .about("Enable required transfer memos for token account")
+                .arg(
+                    account_or_mint_address_arg()
```

Comment:
i have mixed feelings about this. on the one hand, id prefer the interface to be consistent with existing functions, and nothing else behaves like this. unfortunately the existing functions are themselves somewhat inconsistent, because (i assume) a lot of the early work on the cli was before atas were adopted as standard, and (i further assume) we cant break backwards compat anytime in the near future

the typical ways to do this are:

accept the mint address and derive the ata address, providing an --address flag for an override (close, balance, address)
accept the account address only (burn, freeze, thaw)
im inclined to say we should go with the latter option for consistency rather than trying to be clever, and save a refactor of the various other commands for another day

Reply:
resolved with e4dc1d8